test_browser/expected_results.ts

```diff
@@ -0,0 +1,8 @@
+type IExpectedResults = Record<string, string>
+
+const derived =
+  "[1000,1001,2000,2002,2004,2006,2007,2008,2011,2012,2013,2019,2021,2026,2027,2030,2031,2032,2034,2035,2037,2039,2040,2043,2046,2048,2051,2052,2053,2055,2056,2058,2086,2090,2091,2092]"
+
+const derived2 = "{\"value\":\"Derived 2 test results\"}"
```

Comment:
Though I theoretically like being able to communicate expected results back to the server, none of the examples currently have expected results, other than "shouldn't throw". If we were to flesh out the examples to be better tests, I think we'd likely want to inline assertions into the files, rather than putting expected results in another file – in part because there is often not just *one* expected result, as the output often includes non-deterministic things like block hashes.

I think all we need is to communicate pass/fail to the server, and ideally to intercept console logs to send to the server.

Reply:
This is a great point––I also like the idea of us refactoring our examples to include assertions (in a different PR). No need for the browser CI tests to check actual/expected––just that there were no throws.